docs/src/administration/users.md

```diff
@@ -1,65 +1,60 @@
+* Only one environment per project can have Production type.
+* Environments can share the same Staging and Development type.
+* After you add or remove a user from a project or an environment type, you must trigger a redeploy to propagate SSH access changes to each environment. You can redeploy by using the CLI command platform `redeploy` or by clicking **Redeploy** in the management console.
 
-To customize who can use SSH, [set the `access` key](/configuration/app/access.md) in your `.platform.app.yaml` file.
 
-## Manage user permissions in the console
+| Environment permission   | Description |
+|------------------------- |-------------|
+|Viewer                    | Viewers can view all environments of this type. This role includes the Viewer permissions.|
+|Contributor               | Contributors can push code and branch all environments of this type
+|Admin                     | This role includes Viewer and Contributor permissions. Admins can change settings and execute actions on all environments of this type. To customize who can use SSH [set the access key](/configuration/app/access.html) in your platform.app.yaml|
 
-From your list of projects, select the project where you want to view or edit user permissions. Click the Settings tab at the top of the page, then click the `Access` tab on the left to see the users in the project and their roles.
+## Managing users
 
-![Project user management screenshot](/images/management-console/settings-project-access.png)
+### Add a user from the console
 
-Selecting a user to edit that user's permissions or delete the user's access to the project entirely.
+1. Go to your console and select the project where you want to change the user permissions.
+2. Under **Settings**, click **Access**.
+Click **+ Add** and add the user details and choose the permissions.
+4. Click **Save**.
+Once you add the user to the project, they receive an invitation email to confirm their details and a registration link
 
-To add a new user, click the `Add` button.
+### Add a user with the CLI
 
-You can either grant the `Project admin` role to the user, which will give them `Admin` access to every environment in the project, or grant specific permissions for each environment type.
-
-After inviting a new user, users without a Platform.sh account receive an invitation email asking to confirm their details and register an account.
-
-In order to push and pull code (or to SSH into one of the project's environments), the user needs to add an SSH key or use the Platform CLI.
-
-If the user already has an account, they receive an email with a link to access the project.
-
-## Manage users permissions with the CLI
-
-You can use the [Platform.sh CLI (Command Line Interface)](/development/cli/_index.md). to fully manage your users and integrate with any automated system.
+You can use the [Platform.sh CLI (Command Line Interface)](/development/cli/_index.md) to fully manage your users and integrate with any automated system.
```

Comment:
I'm not sure about the connection of the part after "and" because it seems with the "and" like it's different than managing users, but is it?

Reply:
I suppose managing users and integrating with automated systems are two different things. Perhaps I should leave the and integrate with any automated system"out? I'm not saying that in order to integrate systems you don't have to deal with user management but I don't think that bit is relevant within this context.

Reply:
I took it out.